README.md

```diff
@@ -180,6 +181,30 @@ Additionally, container image used to deliver this plugin can be set using
 `KUBE_SECONDARY_DNS_IMAGE` environment variable in operator
 deployment manifest.
 
+## kubevirtIpamController
+
+[This controller](https://github.com/maiqueb/kubevirt-ipam-claims)
+allows to support IPAM for user defined networks.
+
+```yaml
+apiVersion: networkaddonsoperator.network.kubevirt.io/v1
+kind: NetworkAddonsConfig
+metadata:
+  name: cluster
+spec:
+  multus: {}
+  kubevirtIpamController: {}
+```
+
+Additionally, container image used to deliver this plugin can be set using
+`KUBEVIRT_IPAM_CONTROLLER_IMAGE` environment variable in operator
+deployment manifest.
+
+Note: This component requires certificates mounted on the controller pods for the webhook to work.
+On non Openshift clusters, the user should manually install a certificate library (e.g. cert-manager [1]).
```

Comment:
```suggestion
On non Openshift clusters, the user should manually install a certificate library (e.g. [cert-manager](https://github.com/cert-manager/cert-manager)).
```

this is a readme written in markdown, not a commit message, or an email.

Reply:
For what it's worth, it is `OpenShift` . That's what we're asked to call it.

Reply:
done